atomic_reactor/plugins/post_pulp_sync.py

```diff
@@ -208,18 +208,22 @@ def run(self):
             if image.pulp_repo not in repos:
                 repo_id = self.create_repo_if_missing(pulp,
                                                       image.pulp_repo,
-                                                      image.repo)
+                                                      image.to_str(registry=False, tag=False))
                 self.log.info("syncing %s", repo_id)
                 pulp.syncRepo(repo=repo_id,
                               feed=self.docker_registry,
                               **kwargs)
                 repos[image.pulp_repo] = repo_id
 
             images.append(ImageName(registry=pulp_registry,
-                                    repo=image.repo))
+                                    repo=image.to_str(registry=False, tag=False),
+                                    tag=image.tag))
```

Comment:
It was using just the repo name, omitting namespace, so an image 'registry:5000/namespace/repo' would render in crane as 'crane:8888/repo' (unless a correct redirect link was created previously)